src/utils/IndexKeyUtils.h

```diff
@@ -165,6 +165,22 @@ class IndexKeyUtils final {
         return val;
     }
 
+    static std::string encodeRank(EdgeRanking rank) {
```

Comment:
Would you move it to `NebulaKeyUtils`? So `NebulaKeyUtils` don't need to depend on `IndexKeyUtils`

Reply:
but IndexKeyUtils also used encodeRank, if move to NebulaKeyUtils, IndexKeyUtils will need to depends on NebulaKeyUtils.

And encode and decode series of functions currently are in IndexKeyUtils, I think it's better to put them together in IndexKeyUtils.

Reply:
`IndexKeyUtils` are used only for index encoding, I suppose both `IndexKeyUtils` and `NebulaKeyUtils` have a method `encodeRank` is clearer. If one depends on another, which will bring more compile time.

Reply:
OK, will change it according to your comments.

Reply:
`encodeRank `can directly call `encodeInt64`